content/millennium/r4/care-provision/care-team.md

```diff
@@ -0,0 +1,140 @@
+
+The following fields are returned if valued:
+
+* [Care Team id](http://hl7.org/fhir/resource-definitions.html#Resource.id){:target="_blank"}
+* [Status (Active)](http://hl7.org/fhir/careteam-definitions.html#CareTeam.status){:target="_blank"}
+* [Category (Longitudinal or Encounter)](http://hl7.org/fhir/careteam-definitions.html#CareTeam.category){:target="_blank"}
+* [Name (Encounter-Level Only)](http://hl7.org/fhir/careteam-definitions.html#CareTeam.name){:target="_blank"}
+* [Subject](http://hl7.org/fhir/careteam-definitions.html#CareTeam.subject){:target="_blank"}
+* [Encounter (Encounter-Level Only)]( http://hl7.org/fhir/careteam-definitions.html#CareTeam.encounter){:target="_blank"}
+* [Participant](http://hl7.org/fhir/careteam-definitions.html#CareTeam.participant){:target="_blank"}
+  * [Role](http://hl7.org/fhir/careteam-definitions.html#CareTeam.participant.role){:target="_blank"}
+  * [Member](http://hl7.org/fhir/careteam-definitions.html#CareTeam.participant.member){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:care_team, :r4) %>
+
+
+## Search
+
+Search for CareTeams that meet supplied query parameters:
+
+    GET /CareTeam?:parameters
+
+_Implementation Notes_
+
+* The [CareTeam.activity] element is not supported and will not be returned. This means that the [CareTeam.activity.detail.status] and [CareTeam.activity.detail.prohibited] modifier elements will not be returned.
+
+* The description of the CareTeam is included in the [CareTeam.text.div] element as ASCII text.
+
+### Authorization Types
+
+<%= authorization_types(provider: true, system: true) %>
+
+### Parameters
+
+ Name         | Required? | Type          | Description
+--------------|-----------|---------------|--------------
+ `_id`        | See notes | [`token`]     | The logical resource id associated with the resource.
+ `patient`    | See notes | [`reference`] | Who care team is for. Example: `patient=12345`
+ `encounter`  | See notes | [`reference`] | Who care team is for. Example: `encounter=98765`
+ `category`   | N         | [`token`]     | The scope of care team being searched for. Examples: `category=assess-team`
+ `status`     | N         | [`token`]     | Indicates the status of the care team
+
+
+Notes:
+
+- The `_id` parameter
+  - This is required if patient or encounter are not provided.
+
+- The `patient` parameter
+  - This is required when `_id` is not provided.
+  - This can be combined with status as per US Core guidelines.
+  - This can be combined with the category parameter.
+
+- The `encounter` parameter
+  - This is required when `_id` is not provided.
+  - This cannot be combined with category or status.
+
+- The `category` parameter
+  - Only supports the codes longitudinal and encounter.
+  - Can only be used with the patient parameter.
+  - The longitudinal and encounter codes are defined by the CareTeam category system.
+
+### Headers
+
+ <%= headers %>
+
+### Example retrieve care team
```

Comment:
only need to add extra to the Example name when there are multiple examples

Reply:
Removed. https://github.com/cerner/fhir.cerner.com/pull/470/commits/1c1aad456a43d4ba5ae6e7fc8620a05d19202987

Reply:
we want to keep this example, we should title it "Example" instead of "Example retrieve care team" though

Reply:
Addressed https://github.com/cerner/fhir.cerner.com/pull/470/commits/60de10bf8259905011fb638e99b4f1607031c126#diff-05cdc79578a87a8b1066e2bca824be5fR102